resources/views/map/device-dependency.blade.php

```diff
@@ -0,0 +1,40 @@
+@extends('layouts.librenmsv1')
+
+@section('title', __('Device Dependency Map'))
+
+@section('content')
+
+@if($node_count)
+<div id="visualization"></div>
+@else
+<div class="alert alert-success" role="alert">@lang('No devices found')</div>
+@endif
+@endsection
+
+@section('scripts')
+<script src="js/vis.min.js"></script>
```

Comment:
This line should be in `@section('javascript')`

Reply:
And also use {{ asset('js/vis.min.js') }}

Reply:
changed to `{{ asset('js/vis.min.js') }}`
changing to `@section('javascript')` has the effect no Map is graphed

Reply:
```blade
@section('javascript')
<script type="text/javascript" src="{{ asset('js/vis.min.js') }}"></script>
@endsection
```
Works for me.

Reply:
ok,. this works on me also.
Changed to it